docs/references/index.md

```diff
@@ -8,7 +8,7 @@ In this category you will find reference documentation for various components of
 
 ### SDK References
 The DFINITY command-line execution environment (dfx) is the primary tool for creating, deploying, and managing the dapps for the Internet Computer platform.
-Explore `dfx` and its subcommands in the [dfx Commands and Envars](cli-reference/index.md) section and check our beloved [dfx.json](dfx-json-reference.md) schema. 
+Explore the IC SDK and its subcommands in the [dfx Commands and Envars](cli-reference/index.md) section and check our beloved [dfx.json](dfx-json-reference.md) schema. 
```

Comment:
This one should probably be reverted because it is explicitly about dfx.

Reply:
done